pinot-spi/src/main/java/org/apache/pinot/spi/stream/StreamConfig.java

```diff
@@ -68,6 +71,7 @@ public enum ConsumerType {
   private final List<ConsumerType> _consumerTypes = new ArrayList<>();
   private final String _consumerFactoryClassName;
   private final String _decoderClass;
+  private final String _decoderErrorsIgnore;
```

Comment:
I've made this a string instead of boolean in case additional options may need adding in the future, happy to change this to a boolean though

Reply:
IMO, the exception handling logic should be implemented per decoder (either fail the ingestion, or log an error but return `null` to skip the record).
Agree with adding a global config independent of decoder to skip the errored records, but a boolean should be enough.

cc @mcvsubbu @siddharthteotia

Reply:
Yes, totally agree with this as well. We discussed adding this flag when I was making a change to ignore an exception in the confluent decoder, we said initially we would use Boolean so happy to change it - cc:  @kishoreg

Reply:
agreed for boolean